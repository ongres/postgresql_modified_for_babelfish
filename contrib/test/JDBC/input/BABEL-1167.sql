CREATE TABLE babel_1167_table (a INT);
go

INSERT INTO babel_1167_table VALUES (111), (222)
go

CREATE PROCEDURE babel_1167_proc AS
IF 1=1
	UPDATE babel_1167_table SET a = 0
ELSE
	UPDATE babel_1167_table SET a = 1
go

EXEC babel_1167_proc
go

SELECT * FROM babel_1167_table
go

DROP PROC babel_1167_proc
go

DROP TABLE babel_1167_table
go
