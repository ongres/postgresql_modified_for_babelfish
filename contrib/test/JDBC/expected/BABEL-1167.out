CREATE TABLE babel_1167_table (a INT);
go

INSERT INTO babel_1167_table VALUES (111), (222)
go
~~ROW COUNT: 2~~


CREATE PROCEDURE babel_1167_proc AS
IF 1=1
	UPDATE babel_1167_table SET a = 0
ELSE
	UPDATE babel_1167_table SET a = 1
go

EXEC babel_1167_proc
go
~~ROW COUNT: 2~~


SELECT * FROM babel_1167_table
go
~~START~~
int
0
0
~~END~~


DROP PROC babel_1167_proc
go

DROP TABLE babel_1167_table
go
